main.js

```diff
@@ -1284,23 +1284,22 @@ function main() {
     });
 
     zwave.on('value removed', function (nodeID, comClass, instance, index) {
-        adapter.log.info('value removed: ' + nodeID + ' comClass: ' + JSON.stringify(comClass) + ' instance ' + instance + ' value: '  + JSON.stringify(index));
+        adapter.log.debug('value removed: ' + nodeID + ' comClass: ' + JSON.stringify(comClass) + ' instance ' + instance + ' value: '  + JSON.stringify(index));
 
-        deleteDevice(nodeID, function() { //why here and not on "node removed" ?
-            //disable exclusion automatically after 5 seconds
-            setTimeout(function(){disableExclusion();}, 5000);
-        });
+        var id = calcName(nodeID, comClass, index, instance);
+        var list = [ id ];
+        delObjects(list);
```

Comment:
Please remove the list variable and change this to `delObjects([id])`.

Reply:
Oki doki :)